scapy/arch/windows/compatibility.py

```diff
@@ -167,9 +167,64 @@ def sndrcv(pks, pkt, timeout = 2, inter = 0, verbose=None, chainCC=0, retry=0, m
         print "\nReceived %i packets, got %i answers, remaining %i packets" % (nbrecv+len(ans), len(ans), notans)
     return plist.SndRcvList(ans),plist.PacketList(remain,"Unanswered")
 
+def sndrcvflood(pks, pkt, prn=lambda s_r: s_r[1].summary(), chainCC=0, store=1, unique=0, timeout=None):
```

Comment:
Again I don't want to add a docstring, as this is used by `srpflood`, `srflood` (there are no docstring too on the linux part.) This function is basically a mix between the windows' `sndrcv `and the linux `sndrcvflood`